zebra-rpc/src/methods/tests/prop.rs

```diff
@@ -493,6 +496,109 @@ proptest! {
         })?;
     }
 
+    /// Test the `get_blockchain_info` response when tip_pool request fails.
+    #[test]
+    fn get_blockchain_info_returns_genesis_when_tip_pool_fails(network in any::<Network>()) {
+        let (runtime, _init_guard) = zebra_test::init_async();
+        let _guard = runtime.enter();
+        let (mut mempool, mut state, rpc, mempool_tx_queue) = mock_services(network.clone(), NoChainTip);
+
+
+        let genesis_block = match network {
+            Network::Mainnet => {
+                let block_bytes = zebra_test::vectors::CONTINUOUS_MAINNET_BLOCKS[&0];
```

Comment:
It might be clearer to use:

```suggestion
                let block_bytes = &zebra_test::vectors::BLOCK_MAINNET_GENESIS_BYTES;
```